TrigScint/src/TrigScint/TrigScintFirmwareHitProducer.cxx

```diff
@@ -0,0 +1,96 @@
+
+#include "TrigScint/TrigScintFirmwareHitProducer.h"
+
+#include <iterator>
+#include <map>
+
+#include "TrigScint/Firmware/hitproducer.h"
+#include "TrigScint/Firmware/objdef.h"
+
+namespace trigscint {
+
+void TrigScintFirmwareHitProducer::configure(
+    framework::config::Parameters &ps) {
+  pedestal_ = ps.getParameter<double>("pedestal");
+  gain_ = ps.getParameter<double>("gain");
+  mevPerMip_ = ps.getParameter<double>("mev_per_mip");
+  pePerMip_ = ps.getParameter<double>("pe_per_mip");
+  inputCollection_ = ps.getParameter<std::string>("input_collection");
+  testCollection_ = ps.getParameter<std::string>("test_collection");
+  inputPassName_ = ps.getParameter<std::string>("input_pass_name");
+  outputCollection_ = ps.getParameter<std::string>("output_collection");
+  verbose_ = ps.getParameter<bool>("verbose");
+  sample_of_interest_ = ps.getParameter<int>("sample_of_interest");
+  if (verbose_) {
+    ldmx_log(info) << "In TrigScintFirmwareHitProducer: configure done!";
+    ldmx_log(info) << "\nPedestal: " << pedestal_ << "\nGain: " << gain_
+                   << "\nMEV per MIP: " << mevPerMip_
+                   << "\nPE per MIP: " << pePerMip_
+                   << "\ninput collection:     " << inputCollection_
+                   << "\ntest collection:	" << testCollection_
+                   << "\nAre we testing:        " << doTest_
+                   << "\nInput pass name:     " << inputPassName_
+                   << "\nOutput collection:    " << outputCollection_
+                   << "\nVerbosity: " << verbose_;
+  }
```

Comment:
I know that this pattern exists on `trunk` currently but we do have better control over the logging now and so Please convert these messages to use `ldmx_log(debug)` instead of `ldmx_log(info)` and remove the `verbose_` check.

(This comment goes for other locations of `verbose_` as well).

Reply:
Done!